mod-circulation/src/main/resources/logback-test.xml

```diff
@@ -20,14 +20,14 @@
         </encoder>
     </appender>
 
-    <logger name="com.intuit.karate" level="DEBUG"/>
+    <logger name="com.intuit.karate" level="INFO"/>
```

Comment:
revert this, its not our module and lets make this change if needed for our local runs.